src/app/games/game-summaries/external-game/external-game.component.html

```diff
@@ -0,0 +1,26 @@
+<mat-card>
```

Comment:
Would prefer this to be renamed to `external-game-summary`, as it is similar to the `game-summary` component.